kolko.py

```diff
@@ -0,0 +1,66 @@
+
+def printer(tab):
+    'print table with positions to choose'    
+    print('{}\n{}\n{}'.format(tab[0],tab[1],tab[2]))
+
+def instert_to_tab(player,n):
+    '''inserting player to table return update table
+        player - X or O
+        n - position number'''    
+    c=0
+    for arr in tab:
+        for i,pos in enumerate(arr):
+            if n==c:
+                arr[i]=player
+            c += 1
+    return tab
+
+def winner(tab):    
+    '''check who win. Return True when anybody win.
+    tab - used_nums'''
+    for arr in way_to_win:
+        if players[0]==tab[arr[0]]==tab[arr[1]]==tab[arr[2]] or players[1]==tab[arr[0]]==tab[arr[1]]==tab[arr[2]]:
+            return True
+
+def draw(tab):
+    '''if draw True
+    tab - used_nums'''
+    return all([ele for ele in tab])
+
+# table to view
+tab = [['1','2','3'],
```

Comment:
A może być tak po prostu numerować od zera i nie bawić się później w `choice - 1`?
Niby można ciągle odejmować tę jedynkę w każdym miejscu, ale pamiętanie o tym jest uporczywe. Lepiej sobie ułatwiać życie. Nikogo to nie zaboli jesli użytkownik gry poda 0 zamiast 1.

Reply:
Zmienna o nazwie `tab` za wiele nie mówi o tym co się w niej znajduje. Nazwa sugeruje że jest to skrót od tablicy. Żeby ją dobrze nazwać zadaj sobie pytanie: co ona przechowuje?

Reply:
w kolko_pve  zmieniłem z 1 na 0 stało się to drażniące. Trzeba było nie dość że pamiętać od dostosowaniu się do tego złego rozwiązania.